apps/etterlatte-oppdater-behandling/src/test/kotlin/regulering/ReguleringsforespoerselRiverTest.kt

```diff
@@ -216,4 +216,37 @@ internal class ReguleringsforespoerselRiverTest {
             vedtakServiceMock.hentAlleSaker("Regulering2023", 10, emptyList(), SakType.BARNEPENSJON)
         }
     }
+
+    @Test
+    fun `ignorerer spesifiserte saker`() {
+        val melding =
+            JsonMessage.newMessage(
+                mapOf(
+                    ReguleringHendelseType.REGULERING_STARTA.lagParMedEventNameKey(),
+                    DATO_KEY to LocalDate.now(),
+                    KJOERING to "Regulering2023",
+                    ANTALL to 10,
+                    SPESIFIKKE_SAKER to listOf<Long>(),
+                    SAK_TYPE to SakType.BARNEPENSJON.name,
+                ),
+            )
+        val vedtakServiceMock =
+            mockk<BehandlingService>(relaxed = true).also {
+                every { it.hentAlleSaker(any(), any(), any(), any()) } returns
+                    Saker(
+                        listOf(
+                            Sak("Id1", SakType.BARNEPENSJON, 17003, "enhet1"),
+                        ),
+                    )
+            }
+        val featureToggleService =
+            mockk<FeatureToggleService>().also { every { it.isEnabled(any(), any()) } returns true }
```

Comment:
Dra dette ut av funksjonen. Unødvendig å lage en ny instans av disse klassene for _hver eneste_ testfunksjon